packages/fxa-settings/src/pages/Signup/PrimaryEmailVerified/index.test.tsx

```diff
@@ -22,20 +22,21 @@ describe('PrimaryEmailVerified page', () => {
   //   bundle = await getFtlBundle('settings');
   // });
 
-  it('renders the page as expected when the user is signed in', () => {
-    render(<PrimaryEmailVerified />);
+  it('renders the page as expected is signed in', () => {
```

Comment:
nit: reads like a word is maybe missing? maybe change to "...expected _if_ signed in"?